Minio.Functional.Tests/Minio.Functional.Tests.csproj

```diff
@@ -1,43 +1,21 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
-    <RuntimeIdentifiers>
-      ubuntu.16.04-x64;linuxmint.17.3-x64;win10-x64
-    </RuntimeIdentifiers>
-    <PackageId>$(MSBuildProjectName)</PackageId>
-
-
+    <IsPackable>False</IsPackable>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release.Net452|AnyCPU'" />
+  <PropertyGroup Label="Signing">
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <AssemblyOriginatorKeyFile>..\Minio.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release.Core|AnyCPU'" />
-  
-  <Choose>  
-        <When Condition=" '$(Configuration)'=='Debug' ">
-         <ItemGroup>
-            <ProjectReference Include="..\Minio.Core\Minio.Core.csproj" />
-          </ItemGroup>
-          </When>
-        <When Condition=" '$(Configuration)'=='Release.Core' ">
-          <ItemGroup>
-            <ProjectReference Include="..\Minio.Core\Minio.Core.csproj" />
-          </ItemGroup>
-        </When>
-        <When Condition=" '$(Configuration)'=='Mint' ">   
```

Comment:
@niemyjski , can you leave the Mint Configuration here as is - our automated tests need to reference the published version of Minio.NetCore

Reply:
I'll get this in but I'm unable to rebase, it's complaining when I try, you are able to squash and merge when merging the pr (it's a drop down option)